lib/entity.js

```diff
@@ -10,16 +9,39 @@ function Entity (entity) {
   this.type = entity.type || 'Unknown';
   this.name = entity.name || this.id;
 
-  this.experience = 0;
+  this.EXP = 0;
+  this.statPoints = 15;
 
-  this.equipment = {
```

Comment:
Scanned over the rest of the file; is `equipment` just being removed?  Same feedback as earlier comment.

Reply:
Didn't intend to remove equipment completely, that may have been something I was trying to accomplish but meant to undo.